src/components/AnimatedContainer.tsx

```diff
@@ -133,6 +134,7 @@ export function AnimatedContainer({
       pointerEvents={isVisible ? 'box-none' : 'none'}
       {...panResponder.panHandlers}>
       {children}
-    </Animated.View>
+    </Animated.View> :
+    null
```

Comment:
Returning `null` breaks the hiding animation completely

Reply:
What are the options for fixing that issue. It look pretty bad for the users.